inflect/__init__.py

```diff
@@ -219,7 +219,7 @@ def make_pl_si_lists(
     "ox": "oxen",
     "cow": "cows|kine",
     "graffito": "graffiti",
-    "octopus": "octopuses|octopodes",
+    "octopus": "octopuses|octopodes|octopi",
```

Comment:
This change is unrelated and [technically wrong](https://oceanconservancy.org/blog/2022/02/01/plural-octopus/). I'd prefer to omit it, but if you want to advocate for adding it, let's do that in a separate issue/PR.

Reply:
I agree! Let's omit it in this case